plugin/grpctrace/interceptor_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I was just playing around a bit to see how to test this part. Should still wait on #684 and then merge these tests with those.